Header header
uint32 id
float32 battery_status
float32 link_quality
float32 altimeter
float32 mag_x
float32 mag_y
float32 mag_z
float32 gyro_x
float32 gyro_y
float32 gyro_z
float32 acc_x
float32 acc_y
float32 acc_z
uInt16 motor_m1
uInt16 motor_m2
uInt16 motor_m3
uInt16 motor_m4
float32 roll
float32 pitch
float32 yaw